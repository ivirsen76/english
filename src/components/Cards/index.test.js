/* global describe, it, expect */
import React from 'react';
import { shallow } from 'enzyme';
import { toJson } from '@ieremeev/jest-setup';
import { Cards } from './index';


describe('<Cards />', () => {
    it('Should render table', () => {
        const data = [
            { id: 1, text: 'Tree', translate: 'Дерево' },
        ];

        const wrapper = shallow(<Cards data={data} />);
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});
