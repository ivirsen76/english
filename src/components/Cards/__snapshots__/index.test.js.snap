// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Cards /> Should render table 1`] = `
<Table
  className="ui compact table"
  columns={Object {}}
  data={Object {}}
  hideBottomPaginator={false}
  noDataMessage="There is no data"
  perPage={50}
  showRowNumber={true}
/>
`;
